stories/assembly-aap-gcp-monitoring-logging.adoc

```diff
@@ -5,14 +5,14 @@ ifdef::context[:parent-context: {context}]
 
 :context: aap-gcp-monitor-log
 
-You can send metrics to the {GCP} monitoring system to be visualized in the {GCP} UI. 
-{PlatformNameShort} from {GCPMarket} metrics and logging are disabled by default as there is a cost to send these metrics to GCP. 
+You can send metrics to the {GCP} monitoring system to be visualized in the {GCP} UI.
+{PlatformNameShort} from {AAPonGCP} metrics and logging are disabled by default as there is a cost to send these metrics to GCP.
```

Comment:
Apologies. I ought to have been clearer. You only need {AAPonGCP}, it replaces all of {PlatformNameShort} on {GCPMarket} and is much quicker to type

Reply:
Try 2!